cypress/e2e/playground.cy.ts

```diff
@@ -0,0 +1,87 @@
+const getIframeBody = () => {
+    return cy
+      .get('iframe[id="FCL_IFRAME"]')
+      .its('0.contentDocument.body').should('not.be.empty')
+      .then(cy.wrap)
+  }
+  
+describe('Flow-Playground frontend tests', () => {
+
+    beforeEach(() => {
+        cy.viewport('macbook-16')
+        cy.visit('http://localhost:3000/')
+        cy.waitForReact(1000, '#root')
+        
+    })
+
+    it('deploys a contract', () => {
+        cy.react('AccountList').children().last().children().should('have.length', 5).first().click()
+        cy.react('ActionButton', {props: {type: 1}}).contains('Deploy').click()
+        cy.react('ActionButton', {props: {type: 1}}).contains('Redeploy')
+        cy.react('RenderResponse', {props: {resultType: 'CONTRACT'}}).should('exist')
+    })
+
+    it('sends a transaction', () =>{
+        cy.react('MenuList', {props: {title: 'Transaction Templates'}}).children().first().contains('Transaction')
+        cy.get('div[title=Transaction]').click()
+        cy.react('EditorTitle', {props: {type: 2}}).contains('Transaction Template')
+        // Ensure action button is disabled when contract not deployed yet
+        cy.react('ActionButton', {props: {type: 2, active: false}}).contains('Send')
```

Comment:
cypress-react-selector looks pretty cool, but ideally we should test the DOM in these e2e tests. For example, in this case the disabled attribute never gets set on the button, but instead an empty function gets called. 

Using data attributes is also a nice way to keep tests valid even as component structure changes. (https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)

We could look at some helper methods or page objects, but just using data-test attributes would be a great start.  Here's an example:

```
    cy.get('[data-test="action-button"]').click();

    cy.get('[data-test="action-button"]').should('be.disabled');
    cy.get('[data-test="control-panel-status-message"]').should(
      'have.text',
      'Please, wait...',
    );

    cy.get('[data-test="action-button"]').should('be.enabled');
    cy.get('[data-test="control-panel-status-message"]').should(
      'not.have.text',
      'Please, wait...',
    );
```